### PR TITLE
Create frontend reference page for index

### PR DESCRIPTION
**Pull request by maxgds**
https://trello.com/c/fZDCTHQy/740-3-build-mockup-of-index-page-skeleton

Builds the index page skeleton - CSS, JS and markup done, but no real data plugged in. Built to designs seen here: 
https://drive.google.com/open?id=1yUFu4VaRkyEX31QsrWh0V37zh2EPudif

Scope of this does not include the table which will be built using an existing helper and then enhanced.

### CODE REVIEW

File: app/assets/javascripts/core/filtertoggle.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  document.getElementsByClassName("filters-control__link--hide")[0].addEventListener(
+    'click', function() {toggle("hide");}, false
+  );
+  document.getElementsByClassName("filters-control__link--show")[0].addEventListener(
+    'click', function() {toggle("show");}, false
+  );
+}
+
+// Initialise
+var filter = new FilterToggle();
```
Comment (cbaines) [inline app/assets/javascripts/core/filtertoggle.js]: Is this variable used from anywhere?

Comment (maxgds) [reply app/assets/javascripts/core/filtertoggle.js]: nope, good spot, will remove it

File: app/assets/stylesheets/application.scss [hunk 2]
```diff
@@ -14,6 +14,17 @@ $govuk-global-styles: true;
...
 // local views
 @import "metrics/show";
+@import "content/index";
+
+.visually-hidden {
+  @include govuk-visually-hidden;
+}
+
+.gem-c-environment-tag {
+  @media (max-width: 360px) {
```
[the remark this answers is not in the document]
Comment (maxgds) [reply app/assets/stylesheets/application.scss]: It doesn't look great on a big mobile at the smaller size, but on a smaller mobile the contents are overlapping, so it seemed valid to make an exception here, so I'm going to keep it.

File: app/assets/stylesheets/application.scss [hunk 3]
```diff
@@ -14,6 +14,17 @@ $govuk-global-styles: true;
 
 // local views
 @import "metrics/show";
+@import "content/index";
+
+.visually-hidden {
```
[the remark this answers is not in the document]
Comment (maxgds) [reply app/assets/stylesheets/application.scss]: fixed

File: app/views/development/index.html.erb [hunk 4]
```diff
@@ -1,178 +1,130 @@
...
-    <%= render "components/glance-metric", {
-      name: "User satisfaction score",
-      figure: "83",
-      measurement_display_label: "%",
-      context: "Positive feedback, out of 498 user responses",
-      trend_percentage: 3.2,
-      period: "Apr 2018 to Mar 2018"
-    } %>
-  </div>
+  <div class="govuk-grid-column-full">
```
[the remark this answers is not in the document]
Comment (maxgds) [reply app/views/development/index.html.erb]: Done

File: app/assets/stylesheets/application.scss (hunk 2 above)
[the remark this answers is not in the document]
Comment (maxgds) [reply app/assets/stylesheets/application.scss]: I've had to wrap and shrink the gov.uk text as well which doesn't look great, but is necessary to stop the overlap on iphone 5se size. Definitely seems best to constrain it to only the mobiles that need it.

[the remark this answers is not in the document]
Comment (alex-ju) [reply app/assets/stylesheets/application.scss]: Haven't tested on mobile, but I trust you're doing it for a good reason :)

**Pull request merged**